examples/http-compression/http_compression.bal

```diff
@@ -0,0 +1,98 @@
+import ballerina/http;
+import ballerina/log;
+
+endpoint http:Listener listenerEndpoint {
+    port: 9090
+};
+
+// Since compression behaviour of the service is set as COMPRESSION_AUTO, entity body compression is done according
+// to the scheme indicated in `Accept-Encoding` request header. When the header is not present or the
+// header value is "identity", compression is not performed.
+@http:ServiceConfig {
+    compression: {
+        enable: http:COMPRESSION_AUTO
+    }
+}
+service<http:Service> autoCompress bind listenerEndpoint {
+    @http:ResourceConfig {
+        path: "/"
+    }
+    invokeEndpoint(endpoint caller, http:Request req) {
+        caller->respond({ "Type": "Auto compression" }) but {
+            error e => log:printError("Error sending response", err = e)
+        };
+    }
+}
+
+// COMPRESSION_ALWAYS will guarantee a compressed response entity body. Compression scheme is set to the
```

Comment:
Avoid future tense. The word "will" usually indicates that the sentence is in future tense. Instead say "COMPRESSION_ALWAYS guarantees a compressed..."